src/features/lookup_table/use_cases/create_lookup_table.py

```diff
@@ -0,0 +1,9 @@
+from authentication.access_control import DEFAULT_ACL, access_control
+from authentication.models import AccessLevel, User
+from domain_classes.lookup import Lookup
+from storage.internal.lookup_tables import insert_lookup
+
+
+def create_lookup_table(name: str, table: Lookup, user: User) -> None:
```

Comment:
```suggestion
def create_lookup_table_use_case(name: str, table: Lookup, user: User) -> None:
```
Follow naming convention (other features' use cases have this suffix)

Reply:
fixed